concepts/05 UI Components/DropDownBox/00 Getting Started with DropDownBox/25 Customize Appearance.md

```diff
@@ -0,0 +1,99 @@
+Specify the [height](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#height) and [width](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#width) properties to change the size of the DropDownBox text field. If you want to resize the drop-down field, use the [dropDownOptions](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#dropDownOptions) object.
+
+To specify one of the predefined styles for the DropDownBox text field, use the [stylingMode](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#stylingMode) property. To customize the field appearance, specify the [fieldTemplate](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#fieldTemplate).
+
+Use the [placeholder](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#placeholder) property to give users a hint about what they should type in the DropDownBox text field. You can also use the [label](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#label) property for this purpose. If you specify the [label](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#label) property, set the [labelMode](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#labelMode) property to one of the following values:
+
+- *"static"*    
+The component displays the label above the input field.
+
+- *"floating"*    
+The component uses the label as a placeholder, but when the editor gets focus, the label moves to the position above the input field.
+
+- *"hidden"*    
+The label is hidden.
+
+In this tutorial, the component also uses the [label](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#label) as a [placeholder](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#placeholder), because the [labelMode](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#labelMode) property is set to *"floating"*.
```

Comment:
English has different punctuation rules compared to Russian.

```suggestion
In this tutorial, the component also uses the [label](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#label) as a [placeholder](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#placeholder) because the [labelMode](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#labelMode) property is set to *"floating"*.
```